### PR TITLE
Add tables for all equipements

### PR DESCRIPTION
**Pull request by FranckLecuyer**

### CODE REVIEW

**Review by etiennehomer — CHANGES_REQUESTED** [review]

File: src/utils/rest-api.js [hunk 1]
```diff
@@ -246,43 +230,121 @@ export function fetchThreeWindingsTransformers(
...
-        `Fetching equipments of study '${studyName}' of user '${userId}' with substations ids '${substationsIds}'...`
+        `Fetching equipments '${equipmentType}' of study '${studyName}' of user '${userId}' with substations ids '${substationsIds}'...`
     );
-    const fetchAllEquipmentsUrl =
+    const fetchEquipmentsUrl =
         getStudyUrl(studyName, userId) +
-        '/network-map/all' +
+        '/network-map/' +
+        equipmentPath +
+        '' +
```
[the remark this answers is not in the document]
Comment (FranckLecuyer) [reply src/utils/rest-api.js]: Done

File: src/components/study-pane.js [hunk 2]
```diff
@@ -411,6 +454,22 @@ const StudyPane = (props) => {
                         values[0].threeWindingsTransformers
                     );
                     network.updateGenerators(values[0].generators);
+                    network.updateLoads(values[0].loads);
```
Comment (etiennehomer) [inline src/components/study-pane.js:457]: Could be :
updatedEquipments.then((values) => {
                    network.updateSubstations(values.substations);
                    ...

Comment (FranckLecuyer) [reply src/components/study-pane.js:457]: No, because a further US wil concern the lazy loading of a study ...

Comment (etiennehomer) [reply src/components/study-pane.js:457]: ok I see.

**Review by etiennehomer — APPROVED** [review]

**Pull request merged**